components/brave_shields/common/brave_shield_constants.h

```diff
@@ -38,6 +40,46 @@ const char kObsoleteFingerprinting[] = "fingerprinting";
 const base::FilePath::CharType kCustomSubscriptionListText[] =
     FPL("list_text.txt");
 
+constexpr webui::LocalizedString kLocalizedStrings[] = {
```

Comment:
Is this file used by both android and desktop? I haven't looked, but If so, we should wrap this so that it isn't compiled on android as it already has its own strings.

Reply:
doesnt seem to be used in android with a quick ripgrep search of the header file. added flag.